### ISSUE TITLE
.Net 6 Support

### ISSUE THREAD

**Issue by Kylar182** [dotnet/MobileBlazorBindings#352]
Since .Net 5 for this project will seemingly never be supported, I'd like to ask when .Net 6 will be supported (Preview version, etc) and use this thread to track it's release. I know of the Blazor Desktop support in .NET 6 MAUI but I'm looking for .NET 6 Mobile, the original reason for this project and it's namesake.

**Comment by Eilon** [comment]
Hi @Kylar182 thanks for opening this issue! I didn't realize that we don't have an issue tracking this work!

**Comment by Kylar182** [comment]
> Hi @Kylar182 thanks for opening this issue! I didn't realize that we don't have an issue tracking this work!

Feel like giving us an update? One of the main things about .Net 5 for Blazor was the render speed increases. Since we didn't get that on this project because Xamarin was the stick in the mud of .Net 5, I'd really like to see some .Net 6 action.

**Comment by Dreamescaper** [comment]
@Kylar182 
Problem is that Xamarin.Forms does not support neither .NET 5, nor .NET 6. 
So this project will have to migrate to MAUI to be able to use .NET 6. 

MAUI is on early previews stage now, so I don't think any progress will happen here until MAUI release (at least RC, not sure).

**Comment by Eilon** [comment]
.NET MAUI preview 4 is almost out the door as part of .NET 6 Preview 4, and that will have the official BlazorWebView control for Android/iOS/MacCatalyst/WinUI. I also just got a super awesome desktop PC that will let me work much faster ("only" 128GB of RAM 😁) so I should be freed up to get more MBB stuff done now!

**Comment by Kylar182** [comment]
> .NET MAUI preview 4 is almost out the door as part of .NET 6 Preview 4, and that will have the official BlazorWebView control for Android/iOS/MacCatalyst/WinUI. I also just got a super awesome desktop PC that will let me work much faster ("only" 128GB of RAM 😁) so I should be freed up to get more MBB stuff done now!

Yeah I was following https://github.com/dotnet/maui/issues/794#issuecomment-847187387

**Comment by Eilon** [comment]
The PR https://github.com/dotnet/MobileBlazorBindings/pull/409 from @Dreamescaper is now merged! You should be able to clone and build from this repo and see MobileBlazorBindings running on .NET MAUI and .NET 6! Thank you @Dreamescaper for making this possible!
